src/main/java/seedu/address/model/CommandHistoryManager.java

```diff
@@ -6,6 +6,7 @@
  * Keeps track of the valid commands that have been executed in Alfred, and allows command history navigation.
  */
 public class CommandHistoryManager implements CommandHistory {
+    private static final int capacity = 50;
```

Comment:
capacity should be CAPACITY since it's a static final variable.

Reply:
Yup you're right, fixing this now :)